packages/storybook8/stories/Components/MessageStatusIndicator/index.stories.tsx

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { MessageStatusIndicator as MessageStatusIndicatorComponent } from '@azure/communication-react';
+import { Meta } from '@storybook/react';
+import { hiddenControl } from '../../controlsUtils';
+import { MessageStatusIndicatorExample } from './snippets/MessageStatusIndicator.snippet';
+export { MessageStatusIndicator } from './MessageStatusIndicator.story';
+
+export const MessageStatusIndicatorSnippetDocsOnly = {
+  render: MessageStatusIndicatorExample
+};
+
+const meta: Meta = {
+  component: MessageStatusIndicatorComponent,
+  title: 'Components/Message Status Indicator',
+  argTypes: {
+    // Hide default controls
+    readCount: hiddenControl,
+    onToggleToolTip: hiddenControl,
+    remoteParticipantsCount: hiddenControl,
+    styles: hiddenControl,
+    strings: hiddenControl
+  },
+  args: {
+    status: 'delivered',
+    deliveredTooltipText: 'Delivered',
+    sendingTooltipText: 'Sending',
+    seenTooltipText: 'Seen',
+    readByTooltipText: 'Read by',
+    failedToSendTooltipText: 'Failed to send'
```

Comment:
where are these `args` type being defined? For example, `status` is a option control

Reply:
`status` is on the MessageStatusIndicatorProps
The rest match the names in our `MessageStatusIndicatorStrings`

Reply:
Updated the args with display name